app/models/manageiq/providers/base_manager.rb

```diff
@@ -17,5 +19,11 @@ def ext_management_system
     def refresher
       self.class::Refresher
     end
+
+    private
+
+    def require_credentials
+      errors.add(:authentications, 'You must enter credentials.') if missing_credentials?
```

Comment:
We are already doing that, we are validating hostname, that was moved to authentications. So I guess it's ok?

The check right now is, that the default authentication exists and it has a hostname. This check should just add checking credentials on this default authentication, which are marked as required in the UI, so it should be ok.